### PR TITLE
FRONT: Add Login Page & change Call api to utils

### PR DESCRIPTION
**Pull request by badidu09**
Add Login Page & change Call api to utils

MINOR

* **Checklist before requesting a review**
- [x] The commits messages follows our guidelines
- [x] I have performed a self-review of my code
- [x] I have documented my code
- [x] Do we need to implement unit tests?

* **In what state is your pull request?**
- [x] Ready to merge / Waiting a reviwer to see my work.
- [ ] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [ ] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of changes does this PR introduce?** (You can choose multiple)
- [ ] Bug fix
- [x] Feature request
- [ ] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **Describe your changes**

FRONT: Add Login Page & change Call api to utils

* **Issue ticket number and link**

## Summary by CodeRabbit

- **New Features**
  - Enhanced login functionality with error handling for failed login attempts.
  - Added user token storage upon successful login.

- **Bug Fixes**
  - Improved error reporting for the login process.

- **Style**
  - Updated the login page's visual design with new positioning and styling of geometric shapes.

- **Refactor**
  - Standardized quote usage in import statements.
  - Adjusted string literals and code indentation for clarity in the registration process.

### CODE REVIEW

File: front/app/login/page.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	const handleSubmit = async (e: { preventDefault: () => void }) => {
+		e.preventDefault();
+	
+		setError("");
+	
+		try {
+			const json = await loginUser(username, password);
+			setLoginSuccess(true);
+			setToken(json.token);
+			document.cookie = `token=${json.token}`;
```
[the remark this answers is not in the document]
Comment (romainpanno) [reply front/app/login/page.tsx:39]: @coderabbitai  how to do that ?

**Review by Vivelis — CHANGES_REQUESTED** [review]

**Review by Vivelis — DISMISSED** [review]

**Review by KitetsuK — DISMISSED** [review]

File: front/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	const handleSubmit = async (e: { preventDefault: () => void }) => {
+		e.preventDefault();
+	
+		setError("");
+	
+		try {
+			const json = await loginUser(username, password);
+			setLoginSuccess(true);
+			setToken(json.token);
+      Cookies.set('token', json.token, { secure: true, sameSite: 'strict' });
```
Comment (Vivelis) [inline front/app/login/page.tsx:40]: Bad indentation

File: front/app/login/page.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+		} catch (error) {
+      handleError(error);
+    }
```
Comment (Vivelis) [inline front/app/login/page.tsx:43]: Bad indentation

**Review by Vivelis — CHANGES_REQUESTED** [review]

File: front/app/login/page.tsx (hunk 2 above)
Comment (badidu09) [reply front/app/login/page.tsx:40]: image

Comment (badidu09) [reply front/app/login/page.tsx:40]: no on my code

**Review by Vivelis — APPROVED** [review]

**Review by KitetsuK — APPROVED** [review]

**Pull request merged**